bug(android): release build failing with tier.md error

```
01:15:06   FAILURE: Build failed with an exception.
01:15:06   
01:15:06   * Where:
01:15:06   Build file 'C:\BuildAgent\work\78a54344b259d3bb\keyman\android\KMAPro\kMAPro\build.gradle' line: 131
01:15:06   
01:15:06   * What went wrong:
01:15:06   A problem occurred evaluating project ':kMAPro'.
01:15:06   > Could not get unknown property 'tier' for extension 'play' of type com.github.triplet.gradle.play.PlayPublisherExtension.
```

https://build.palaso.org/buildConfiguration/KeymanAndroid_Build/525383?buildTab=log&linesState=842&logView=flowAware&focusLine=1599

--- a/android/KMAPro/kMAPro/build.gradle
+++ b/android/KMAPro/kMAPro/build.gradle
@@ -9,6 +9,8 @@ plugins {
 ext.rootPath = '../../'
 apply from: "$rootPath/version.gradle"
 
+String tier = new File('../../TIER.md').getText('UTF-8').trim();
+
 android {
     compileSdk 34
     namespace="com.tavultesoft.kmapro"
@@ -67,7 +69,6 @@ android {
     productFlavors {
     }
 
-    String tier = new File('../../TIER.md').getText('UTF-8').trim();
     applicationVariants.all { variant ->
         // Append tag to app name to help differentiate versions for app testers:
         //  - stable builds have no suffix
@@ -128,7 +129,7 @@ if (env_keys_json_file != null && file(env_keys_json_file).exists()) {
 
         // Deactivate lower conflicting version APKs
         resolutionStrategy.set(com.github.triplet.gradle.androidpublisher.ResolutionStrategy.IGNORE)
-        switch (String.valueOf(tier)) {
+        switch (tier) {
             case 'beta':
                 track.set("beta")
                 println "TIER set to beta"

--- a/oem/firstvoices/android/app/build.gradle
+++ b/oem/firstvoices/android/app/build.gradle
@@ -54,7 +54,6 @@ android {
     productFlavors {
     }
 
-    String tier = new File('../../../TIER.md').getText('UTF-8').trim();
     applicationVariants.all { variant ->
         // Adjust output name to "firstvoices-${VERSION_MD}.apk"
         variant.outputs.all {
@@ -96,7 +95,8 @@ if (env_keys_json_file != null && file(env_keys_json_file).exists()) {
 
         // Deactivate lower conflicting version APKs
         resolutionStrategy.set(com.github.triplet.gradle.androidpublisher.ResolutionStrategy.IGNORE)
-        switch (String.valueOf(tier)) {
+        String tier = new File('../../../TIER.md').getText('UTF-8').trim();
+        switch (tier) {
             case 'beta':
                 track.set("beta")
                 println "TIER set to beta"

